src/main/java/com/iabtcf/decoder/TCString.java

```diff
@@ -27,7 +27,7 @@
 import com.iabtcf.v2.PublisherRestriction;
 
 public interface TCString {
-    public static TCString decode(String consentString) {
+    static TCString decode(String consentString) {
```

Comment:
this needs to be public

Reply:
public is redundant for interface methods